packages/driver/src/cy/multi-domain/index.ts

```diff
@@ -154,6 +154,8 @@ export function addCommands (Commands, Cypress: Cypress.Cypress, cy: Cypress.cy,
 
         // fired once the spec bridge is set up and ready to receive messages
         communicator.once('bridge:ready', () => {
+          // now that the spec bridge is ready, instantiate Cypress with the current app config
+          communicator.toSpecBridge('initialize:cypress', Cypress.config())
```

Comment:
I worry about this because, while our own config is serializable, a user can mutate the config with arbitrary keys and values. What happens if a user does something like this?

```js
Cypress.config('el', document.querySelector('#el'))

cy.switchToDomain(...)
```

It will blow up because there's an unserializable value, right? Might need to be more defensive about it.

Reply:
I hadn't even considered that. I think it will blow up because we don't do any type of preprocessing on `toSpecBridge`. Going to look into this and explore some options.

Reply:
config syncing is now addressed in #20102